src/pages/[id].page.jsx

```diff
@@ -41,17 +43,40 @@ export default function Details() {
   );
 }
 
-export const getStaticPaths = async () => ({
-  paths: [
-    '/accordion',
-    '/dialog',
-    '/disclosure',
-    '/tabs',
-  ],
-  fallback: 'blocking', // indicates the type of fallback
-});
+export async function getStaticPaths() {
+  return {
+    paths: [
+      {
+        params: { id: 'accordion' },
+      },
+      {
+        params: { id: 'dialog' },
+      },
+      {
+        params: { id: 'disclosure' },
+      },
+      {
+        params: { id: 'tabs' },
+      },
+    ],
+    fallback: 'blocking', // indicates the type of fallback
+  };
+}
 
 // pre-render this page on each request using data in props.
-export const getStaticProps = async () => ({
-  props: {},
-});
+export async function getStaticProps({ params }) {
+  try {
+    const { id } = params;
+    // run the airtable function to get the data and save to the details variable.
+    const details = await getComponentDetails(id);
+    // pass that details variable and its data into props to use elsewhere
+    return {
+      props: {
+        componentName: details['Component Name'],
```

Comment:
Nice Job! This was a really neat solution to getting Airtable data into the component.